examples/apps/Django_Panel_Multi/README.md

```diff
@@ -0,0 +1,472 @@
+# Django Apps
+
+As the core user guides including the Introduction have demonstrated, it is easy to display Panel apps in the notebook, launch them from an interactive Python prompt, and deploy them as a standalone Bokeh server app from the commandline. However, it is also often useful to embed a Panel app in large web application, such as a Django web server. Using Panel with Django requires a bit more work than for notebooks and Bokeh servers.
+
+To run this example app yourself, you will first need to install django 2 or 3 (e.g. conda install "django=2").
+
+We will show how to build a simple django apps with 3 apps stockscreener, sliders, sliders_cosine. this app shows how to integrate Panel with a Django view, but there's currently no interaction between the Param and Django models.
+
+### Init
```

Comment:
Looks like a good start but instead of pasting in the actual file contents a high-level description of what's in the different files and what they do is probably preferable.

Reply:
OK, it is my first time...learning

> My only other comment is that the market_data.csv is pretty big and we might want to find a different way to ship that file or include only a subset.

I can easily create a much smaller subset